src/components/Widget.tsx

```diff
@@ -35,6 +35,8 @@ const slideOutRight = keyframes`
   }
 `
 
+export const DialogAnimationLengthMs = 250
```

Comment:
just a heads up im going to be replacing all these timing variables in a followup with design tokens